docs/Developers/Writing_Good_Commits.rst

```diff
@@ -150,6 +150,10 @@ Example:
    If a specific issue may lead to an unintended behaviour from the user
    or from the program it should be considered a bug, and should be
    addresed with ``Fixes``.
+-  If an issue is labelled with ``type/bug`` you should always use ``Fixes``.
+-  Issues that require fixing some typing errors should also be referenced
```

Comment:
no :) typing errors are no bugs imo ;) (except bad referencing of variables, but this is not happening because of our full coverage)